gui/src/launcher.rs

```diff
@@ -206,6 +207,25 @@ async fn check_network_datadir(mut path: PathBuf, network: Network) -> Result<Ne
             )
         }
     })?;
+        if let Some(exe_config) = cfg.internal_bitcoind_exe_config {
```

Comment:
Launcher is an intermediary step to check if an install already exist for the existing network and if none it redirects user to the install process. I believe this piece of code should be done on the loader side.

Reply:
OK sure. Would `Loader::on_load()` be a suitable place, just before it tries to start the Liana daemon in one of the error `match` arms?

Reply:
I think it should be done in the 'start_daemon' method of the loader, that you can rename in 'start_bitcoind_and_daemon'
because if a daemon is already running it means that a bitcoind is also running.

Reply:
Thanks for that. I've made this change in https://github.com/wizardsardine/liana/pull/592/commits/783d93038d5575f880a97fc61e290b5bd96e8fe0.